### DISCUSSION TITLE
Can you use a TLS server authentication (not mutual TLS) listener when using the Topic Operator?

### DISCUSSION THREAD

**Discussion by Otto45** [strimzi/strimzi-kafka-operator#6003] [Q&A] [answered]
Quick question: Can you create and use a listener for TLS Server Authentication (not mutual TLS) when using the Topic Operator? I have been playing around with Strimzi for a few days now, and one scenario I can't seem to get working is creating a listener (internal or external) with server only tls authentication, so setting `tls: true` for the listener but not specifying the `authentication` property for the listener. I am able to get mutual tls authentication working with my python producers and consumers connecting to another listener setup for mutual tls. The topic I am using was created via the Topic Operator using a KafkaTopic CR. I had assumed this scenario is valid with Strimzi, or is it only possible to use mutual TLS when using the Topic Operator to manage topics?

**Comment by scholzj** [comment, marked as answer]
The Topic Operator is using an internal internal listener which is not user configurable and is always using mTLS. It should not matter what do you configure in the listeners in the Kafka custom resource

Just to clarify and make sure we are talking about the same thing -> when you use Strimzi to run the Kafka cluster, you deploy the Topic Operator directly from the `Kafka` custom resource such as in this example: https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/kafka/kafka-persistent.yaml#L38 ... you do not need to install it separately.

**Reply by Otto45** [reply]
Yes, I deploy the Topic Operator directly from the `Kafka` CR. So if I'm understanding things correctly, I have to use mutual TLS if I created the topic using a `KafkaTopic` CR, which means I have to setup my listener to use mutual TLS?

**Reply by scholzj** [reply]
No, if you deployed it through the Kafka CR, you can use whatever you want for your clients and it will have no impact on the Topic Operator. When deployed through the Kafka CR, it is not using any of the listeners you configure there but a completely separate listener which you don't configure at all.

**Reply by Otto45** [reply]
Got it, thanks for the clarification.
